AlertStyles.css

```diff
@@ -0,0 +1,102 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700;900&display=swap');
+
+:root {
+    --primary-color : #c2004b;
```

Comment:
put the common colors, which will be used in other components also, in a file in the root folder, import from there